ompi/mpi/c/win_get_info.c

```diff
@@ -27,6 +27,10 @@
 
 static const char FUNC_NAME[] = "MPI_Win_get_info";
 
+static void _win_info_set (ompi_info_t *info, const char *key, int set)
```

Comment:
Symbols with "_" prefixes are supposed to be the domain of the compiler, not user functions.